examples/pylab_examples/spectrum_demo.py

```diff
@@ -1,32 +1,43 @@
+"""
+========================
+Spectrum Representations
+========================
+
+The plots show different spectrum representations of a sine signal with
+additive noise. A (frequency) spectrum of a discrete-time signal is calculated
+by utilizing the fast Fourier transform (FFT).
+"""
 import matplotlib.pyplot as plt
 import numpy as np
 
 
 np.random.seed(0)
 
-dt = 0.01
-Fs = 1/dt
+dt = 0.01  # sampling interval
+Fs = 1/dt  # sampling frequency
 t = np.arange(0, 10, dt)
+
+# generate noise:
 nse = np.random.randn(len(t))
 r = np.exp(-t/0.05)
-
 cnse = np.convolve(nse, r)*dt
 cnse = cnse[:len(t)]
-s = 0.1*np.sin(2*np.pi*t) + cnse
-
-plt.subplot(3, 2, 1)
-plt.plot(t, s)
 
-plt.subplot(3, 2, 3)
-plt.magnitude_spectrum(s, Fs=Fs)
+s = 0.1*np.sin(2*np.pi*t) + cnse  # the signal
 
-plt.subplot(3, 2, 4)
-plt.magnitude_spectrum(s, Fs=Fs, scale='dB')
+fig, axes = plt.subplots(nrows=3, ncols=2)
 
-plt.subplot(3, 2, 5)
-plt.angle_spectrum(s, Fs=Fs)
+# plot time signal:
+axes[0, 0].plot(t, s)
+axes[0, 0].set_xlabel("Time")
+axes[0, 0].set_ylabel("Amplitude")
 
-plt.subplot(3, 2, 6)
-plt.phase_spectrum(s, Fs=Fs)
+# plot different spectrum types:
```

Comment:
I added titles, but I think it's worthwhile leaving the default axis labels, since it's feature of the spectrum functions.